examples/hotrod/services/frontend/best_eta.go

```diff
@@ -76,9 +77,10 @@ func (eta *bestETA) Get(ctx context.Context, customerID string) (*Response, erro
 	}
 	eta.logger.For(ctx).Info("Found customer", zap.Any("customer", customer))
 
-	if span := opentracing.SpanFromContext(ctx); span != nil {
-		span.SetBaggageItem("customer", customer.Name)
-	}
+	m, _ := baggage.NewMember("customer", customer.Name)
+	bag := baggage.FromContext(ctx)
+	bag, _ = bag.SetMember(m)
```

Comment:
```suggestion
	m, err := baggage.NewMember("customer", customer.Name)
	if err != nil {
		eta.logger.For(ctx).Error("cannot create baggage member", zap.Error(err))
	}
	bag := baggage.FromContext(ctx)
	bag, err = bag.SetMember(m)
	if err != nil {
		eta.logger.For(ctx).Error("cannot set baggage member", zap.Error(err))
	}
```

Reply:
Cool! ignoring `err` costs me here.

Reply:
@yurishkuro after deep investigation, it came out working with the response in the HTTP request. e.g. ```Loading customer      {"service": "customer", "component": "mysql", "trace_id": "63718f14ff8d04607e60d95da54784ae", "span_id": "b60a6202006008bc", "customer_id": "567"}``` here, when I try to use any of the items included say "span_id" or customer.ID or customerID or "mysql", I am getting baggage items out of it. Make sure the service should be "customer".

Reply:
Most insane. when I try to add logs into it, it's still not working. `INFO    customer/database.go:86 Getting customer      {"service": "customer", "component": "mysql", "trace_id": "49af5501b6c5c75acfb1e69d19f1e481", "span_id": "e9e53f530cb6aacc", "customer_name": "Amazing Coffee Roasters"}`

Reply:
after making minor changes, I got sth like below
```
"route.calc.by.customer.sec": {"Amazing_Coffee_Roasters": 0.498, "Japanese_Desserts": 0.486},
"route.calc.by.session.sec": {"1581": 0.498, "7406": 0.486}
```

Reply:
Yes, that's the bug it OTEL - users are not supposed to mangle the values like this in order to put them into baggage.